src/components/Calculator.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import './Calculator.css';
+
+const Utility = () => (
+  <div className="utility">
+    <p>AC</p>
+    <p>+/-</p>
+    <p>%</p>
+  </div>
+);
+
+const LeftSide = () => (
+  <div className="leftSide">
+    <p>÷</p>
+    <p>*</p>
+    <p>-</p>
+    <p>+</p>
+    <p>=</p>
+  </div>
+);
```

Comment:
- [ **OPTIONAL** ] As I mentioned in the previous comment, I think it would be better if you used the button elements in `<button>` tags  for good tags semantics,